app/src/main/java/org/mozilla/fenix/library/recentlyclosed/RecentlyClosedFragmentView.kt

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.library.recentlyclosed
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.core.content.ContextCompat
+import androidx.core.view.isVisible
+import androidx.recyclerview.widget.LinearLayoutManager
+import kotlinx.android.extensions.LayoutContainer
+import kotlinx.android.synthetic.main.component_recently_closed.*
+import mozilla.components.browser.state.state.ClosedTab
+import org.mozilla.fenix.R
+
+interface RecentlyClosedInteractor {
```

Comment:
I think we could move this to the RecentlyClosedFragmentInteractor.kt and rename the implementation to DefaultRecentlyClosedFragmentInteractor.